### ISSUE TITLE
books: edit book data

### ISSUE THREAD

**Issue by bwhicks** [Princeton-CDH/derrida-django#2]
As a record editor, I want to add new data in or edit the following fields so that I can document them in a standard way:

Bibliographic information (Title, Author, Place of publication, Publisher, Publication year, Original publication year, printing year, Other creators including translator, editor, co-author)
Language
Translation (i.e., book is a translation from another language; yes/no)
Subject
Type of publication (book, journal, article)
Extant in PUL JD Library Collection (yes/no)
        If yes:
Finding Aid URL
Derrida shelf mark
Contains annotations (yes/no)
Contains insertions (yes/no)
Digitized (yes/no)
Contains dedication

**Comment by kac3** [comment]
@bwhicks Ben, I noticed that most (all?) of the authors seem to have imported twice, so that every book record has a double author. That's not a big deal, but I tried to delete one (on Lévi-Strauss's Anthropologie Structurale) and was not able to do so.  Help?

**Comment by bwhicks** [comment]
@kac3 Sorry, didn't realize you were testing right now. I'm pushing a fix that should also get the duplicates (which were a known problem but obviously distracting). Once the database is back up, I'll ping this again.

**Comment by kac3** [comment]
@bwhicks No worries, and thanks!

**Comment by bwhicks** [comment]
@kac3 Okay, the database is back. Copyright year has hit a snag (known issue, will sort by Monday), but references are now turning up correctly on book entries!

**Comment by bwhicks** [comment]
@kac3 I've sorted those snags in the import, so if you could email/github me when you're done with interface testing this weekend, I can get the data set cleaned up. Sorry for that... perils of code at 4:00pm on a Friday.

**Comment by kac3** [comment]
@bwhicks I remember that the "original publication information" field was tricky because of the need for BCE and CE, plus the need to note possible discrepancies for when a work was written and when it was published.  All that said, is there a standard format for this field?  And if there isn't, will users be able to effectively filter by date (e.g., works originally published between 1900 and 1920)?

**Comment by kac3** [comment]
@bwhicks I've hit two issues, both of which are new things I realized in the testing process that we will need.

1. There is one more field I would like to add here if possible:
"This edition cited by Derrida" (yes/no)

Since Derrida often owns multiple copies of a book, this allows me(and, eventually, users) to keep track of which ones he cites and which ones are alternate copies.

2. Relatedly, I would ideally like to be able to associate books that are 1 identical copies of the same work, 2) different editions of the same work, 3) translations of the same work, or 4) contain the same book sections.  This is similar to the Book Section - Book relationship, but different because it relates two Books. 

For example (type 3 above), I want to be able to relate Heidegger's Sein und Zeit in German to the English Being and Time and the French Être et Temps.  

I do not think we need to specify the type of relation; we just need to be able to associate the records.

**Comment by bwhicks** [comment]
@kac3 Great. Your point 2 clarifies some questions I had had. It sounds like you don't have section - book as separate entries all the time, only when Derrida owned both? Correct me if I'm wrong on this one.

On issue one, I'll mark that as part of the database changes I'll be making for some of your other feedback.

**Comment by kac3** [comment]
Hi @bwhicks, 

On the Section - Book question in point 2 above, this is (confusingly, I can imagine) a different issue from the Book Section - Book issue we had talked about before.

The issue I raised in point 2 would occur in the following example: Derrida owns a copy of Rousseau's "Profession de foi" (a slender volume containing a single text) and "Oeuvres complètes, tome 3" (a massive volume containing many different texts, including "Profession de foi").  I would like to be able to associate these two records, both of which are Books.  On the site, these will have their own Book View pages and the covers will appear on the Derrida's Library page.

BUT (and here's where things get tricky) there will be a **different** record for the "Profession de foi" that is a Book Section within the Book "Oeuvres complètes, tome 3."  This is the issue we had discussed before.  Here I need a different type of association, because we are not dealing with two Books but a Book Section within a Book.  Book Sections will not appear on the Derrida's Library page and will not have their own Book View page.  We do need them as separate records, however, since Derrida cites the Book Section (and not the Book "Oeuvres complètes").  

Ideally, I could also associate "Profession de foi" the Book Section (in "Oeuvres complètes") with "Profession de foi" the Book to indicate that they are different editions of the same text but without implying that "Profession de foi" the Book Section is contained within "Profession de foi" the Book.  If this proves too tricky we can simply associate "Profession de foi" the Book with "Oeuvres complètes."

Apologies if this is overly explicit, but I just wanted to be clear since this can get confusing!
